### DISCUSSION TITLE
Problem setting up gitmoji

### DISCUSSION THREAD

**Discussion by meatballs** [kovidgoyal/kitty#3868] [Q&A] [answered]
Hi

I'm attempting to add the unicode characters for [gitmoji](https://gitmoji.dev) to my [unicode favourites](https://github.com/meatballs/dotfiles/blob/master/kitty/unicode-input-favorites.conf) - mostly successfully.

However, for one or two, if I copy the character from the gitmoji site and paste into my terminal it looks fine but, when I find the unicode codepoint and enter that, the display is different. 

An example is the coffin emoji - ⚰️ - the unicode for which, I believe is u+26b0. However, that displays with a much simpler black on white 'icon'.

Any ideas where I'm going wrong?

**Comment by kovidgoyal** [comment, marked as answer]
On Mon, Jul 26, 2021 at 07:19:33AM -0700, Owen Campbell wrote:
> Hi
> 
> I'm attempting to add the unicode characters for [gitmoji](https://gitmoji.dev) to my [unicode favourites](https://github.com/meatballs/dotfiles/blob/master/kitty/unicode-input-favorites.conf) - mostly successfully.
> 
> However, for one or two, if I copy the character from the gitmoji site and paste into my terminal it looks fine but, when I find the unicode codepoint and enter that, the display is different. 
> 
> An example is the coffin emoji - ⚰️ - the unicode for which, I believe is u+26b0. However, that displays with a much simpler black on white 'icon'.
> 
> Any ideas where I'm going wrong?

That's a symbol that has default text presentation, if you want to see
it as an emoji you need the character itself followed by ufe0f the emoji
variant selector. See the --emoji-variation option of the unicode input
kitten.

**Reply by meatballs** [reply]
ahhhhhh!!!!! Fantastic. Thank you!
